packages/core/lib/core/validation.js

```diff
@@ -13,7 +13,7 @@ export function isNumberValid(v: any) {
 }
 
 export function assertKeyType(type) {
-  assert(KeypairTypes.find(t => t === type), `Invalid keypair type ${type}. Expected on of ${KeypairTypes.join(',')}`);  
+  assert(!!KeypairTypes.find(t => t === type), `Invalid keypair type ${type}. Expected on of ${KeypairTypes.join(',')}`);  
```

Comment:
linting and I'm not sure `Expected on of ${xx}` makes sense grammatically.

Reply:
It should be `Expected one of ${xx}`. Let me fix that, is that better? Lemme know if you have any other sugestion